google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -608,13 +611,17 @@ private void appendRawPathFromParts(StringBuilder buf) {
   }
 
   /** Adds query parameters from the provided entrySet into the buffer. */
-  static void addQueryParams(Set<Entry<String, Object>> entrySet, StringBuilder buf, boolean verbatim) {
+  static void addQueryParams(
+      Set<Entry<String, Object>> entrySet, StringBuilder buf, boolean verbatim) {
     // (similar to UrlEncodedContent)
     boolean first = true;
     for (Map.Entry<String, Object> nameValueEntry : entrySet) {
       Object value = nameValueEntry.getValue();
       if (value != null) {
-        String name =  verbatim ? nameValueEntry.getKey() : CharEscapers.escapeUriQuery(nameValueEntry.getKey());
+        String name =
```

Comment:
I preferred the original one line version here.

Reply:
I don't disagree, but this with what running `mvn com.coveo:fmt-maven-plugin:format` yields .